recipes/libx265/all/conanfile.py

```diff
@@ -54,15 +56,18 @@ def configure(self):
 
     def build_requirements(self):
         if self.options.assembly:
-            self.build_requires("nasm/2.14")
+            self.build_requires("nasm/2.15.05")
 
     def requirements(self):
         if self.options.get_safe("with_numa", False):
             self.requires("libnuma/2.0.14")
 
+    def validate(self):
+        if self.settings.arch not in ["x86", "x86_64"]:
+            raise ConanInvalidConfiguration("Only x86/x86_64 supported")
```

Comment:
More or less the same remark than OpenColorio, upstream CMakeLists handles different architectures, this message is misleading: the recipe doesn't injects proper CMake variables for the moment, but others architectures are supported.

Reply:
Yes, you are right. Plesae @SSE4 , as you are also creating some PRs related to M1, keep in mind this issue as well, we can write something like _"Current recipe doesn't support XXX (yet)"_, will it be more accurate?

Reply:
Yes, it's important to say that the recipe, not the library, doesn't yet support cross-build (which is not completly true by the way, I'm pretty sure that cross-build to x86_64 works).